### ISSUE TITLE
Ticking Tile Crash

### ISSUE THREAD

**Issue by grissly** [LatvianModder/FTBUtilities#791]
+ See if the issue isn't already reported / suggestion been made by looking up some keywords.
+ If it is, don't comment that you have the same issue, just press :+1:
+ Bugs and Suggestions will only be accepted and implemented/fixed for latest minecraft version, unless it's compatible with older one.
+ Do not add versions or "[Bug]" or "[Suggestion]" in issue title, I will tag them.

**Comment by grissly** [comment]
I'm still trying to figure out what happened, but I got a ticking tile entity crash twice (after reverting backup). I posted on actually additions page since the affected block was from that mod, but they told my it was the fault of ftblib. I noticed a method in the second line of the stack trace was from ftblib, so I'm posting the logs here. Sorry I can't be more helpful as I don't really know what I'm looking at.

https://github.com/Ellpeck/ActuallyAdditions/issues/1117

[crash-2018-06-16_21.27.42-server.txt](https://github.com/LatvianModder/FTBUtilities/files/2111904/crash-2018-06-16_21.27.42-server.txt)
[crash-2018-06-15_15.12.31-server.txt](https://github.com/LatvianModder/FTBUtilities/files/2111905/crash-2018-06-15_15.12.31-server.txt)
